### PR TITLE
feat: adds the experimental toggle for Solana 

### PR DESCRIPTION
**Pull request by zone-live**
## **Description**

This PR sdds a new experimental toggle for Solana accounts. Everything is code fenced for Flask build.

Screenshot 2024-10-31 at 11 50 38

## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to this page...
2.
3.

## **Pre-merge author checklist**

- [ ] I've followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Extension Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: app/_locales/en/messages.json [hunk 1]
```diff
@@ -5224,6 +5224,16 @@
     "message": "Contact the creators of $1 for further support.",
     "description": "This is shown when the insight snap throws an error. $1 is the snap name"
   },
+  "solanaSupportSectionTitle": {
+    "message": "Solana"
+  },
+  "solanaSupportToggleDescription": {
+    "message": "Turning on this feature will give you the option to add a Solana Account to your MetaMask Extension derived from your existing Secret Recovery Phrase. This is an experimental Beta feature, so you should use it at your own risk.",
+    "description": "$1 is the link to a product feedback form"
```
Comment (pedronfigueiredo) [inline app/_locales/en/messages.json:5232]: Change description?

File: ui/pages/settings/experimental-tab/experimental-tab.component.tsx [hunk 2]
```diff
@@ -369,6 +373,53 @@ export default class ExperimentalTab extends PureComponent<ExperimentalTabProps>
...
+        {this.renderToggleSection({
+          title: t('solanaSupportToggleTitle'),
+          description: t('solanaSupportToggleDescription', [
+            <a
+              key="btc-account-feedback-form__link-text"
+              href={SurveyUrl.BtcSupport}
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              {t('form')}
```
Comment (pedronfigueiredo) [inline ui/pages/settings/experimental-tab/experimental-tab.component.tsx:401]: update this key?

File: ui/pages/settings/experimental-tab/experimental-tab.component.tsx [hunk 3]
```diff
@@ -369,6 +373,53 @@ export default class ExperimentalTab extends PureComponent<ExperimentalTabProps>
...
+          marginBottom={2}
+          fontWeight={FontWeight.Bold}
+        >
+          {t('solanaSupportSectionTitle')}
+        </Text>
+        {this.renderToggleSection({
+          title: t('solanaSupportToggleTitle'),
+          description: t('solanaSupportToggleDescription', [
+            <a
+              key="btc-account-feedback-form__link-text"
```
Comment (pedronfigueiredo) [inline ui/pages/settings/experimental-tab/experimental-tab.component.tsx:396]: update this key?

File: app/_locales/en/messages.json (hunk 1 above)
Comment (zone-live) [reply app/_locales/en/messages.json:5232]: True, "description" can be removed 👍🏼

**Review by pedronfigueiredo — APPROVED** [review]

**Review by ulissesferreira — APPROVED** [review]

**Pull request merged**